### PR TITLE
New metaproteomics workflow for F1000 Query Tabular paper 

### PR DESCRIPTION
**Pull request by pravs3683**

### CODE REVIEW

**Comment by bgruening** [comment]
@pravs3683 awesome! Works for me locally! Do you want to include more tests, or do you think one is enough?

**Comment by pravs3683** [comment]
Thanks @bgruening . I have added few more test. Two more separate workflows will be coming up soon after this.

**Comment by bgruening** [comment]
Awesome! Keep them coming :)

File: training/metabolomics/F1000_Metabolomics_Query_Tabular/F1000_Metabolomics_Query_Tabular_Mass_Adjustment.ga [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+        "top": 365.5
+      },
+      "post_job_actions": {
+        "HideDatasetActionoutput_html": {
+          "action_arguments": {},
+          "action_type": "HideDatasetAction",
+          "output_name": "output_html"
+        }
+      },
+      "tool_id": "testtoolshed.g2.bx.psu.edu/repos/eslerm/vkmz_qt/vkmz_qt/1.3.1",
```
Comment (bgruening) [inline training/metabolomics/F1000_Metabolomics_Query_Tabular/F1000_Metabolomics_Query_Tabular_Mass_Adjustment.ga]: same here .... please make sure this is running against EU and let us know which tools you need.

**Comment by bgruening** [comment]
With the last workflow which you added I also get a `toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastp_wrapper/0.3.1` not found error. Which is strange as we have installed this tool, maybe its just my bad internet connection.

**Comment by pravs3683** [comment]
Thanks Bjorn, I will check with the developer and re-submit the workflow
for the metabolomics part. Meanwhile, can we please have Metaproteomics and
Proteogenomics workflows submitted?

Thanks,
Praveen

**Comment by bgruening** [comment]
Ok, do you want to remove the non-functional workflows from this PR. Next time just create separate PRs, the first one was perfect and I was ready to merge it :) 

Thanks again!

**Comment by pravs3683** [comment]
Sure. I will do that. I will do a separate PR for each one. Thanks Bjorn

**Pull request closed without merging**